controllers/topics.controllers.js

```diff
@@ -5,4 +5,8 @@ exports.getTopics = (req, res, next) => {
     .then((output) => {
       res.status(200).send({ topics: output });
     })
+    .catch((err) => {
+      console.err(err);
+      res.status(500).send({ err: "Unable to retrieve topics!" });
```

Comment:
the 500 error should not be in the .catch block this should be an external error handler in app.js
https://notes.northcoders.com/courses/js-back-end/error-handling
see the second example in Error Handling Middleware in the notes to refer to 😊

Reply:
and then think about what we could use in this catch block to pass the error along....

Reply:
Again, Inna, the 500 error handler should be in app.js as per the linked notes. make this change before you merge